searchwidget/src/main/java/com/harsh/searchwidget/Model/AnalyticsModel.kt

```diff
@@ -0,0 +1,11 @@
+package com.harsh.searchwidget.Model
+
+/**
+ * A Kotin Data class that holds data pertaining to Analytics
+ * Removes verbosity introduces due to getters and setters in Java
+ */
+
+data class AnalyticsModel(var xSearchId: String?,
+                     var isXSearchClick: Boolean,
+                     var xSearchClickPosition: String?,
+                     var isXSearchConversion: Boolean)
```

Comment:
```suggestion
data class AnalyticsModel (
    var xSearchId: String?,
    var isXSearchClick: Boolean,
    var xSearchClickPosition: String?,
    var isXSearchConversion: Boolean
)
```

Follow this everywhere.